src/utils/destroy-backbone-view.js

```diff
@@ -11,6 +12,7 @@ export default function destroyBackboneView(view) {
     triggerMethodOn(view, 'before:detach', view);
   }
 
+  dom.removeEl(view.el);
```

Comment:
Is this going to work?  Might be unnecessary?  Seems like if someone is using a vanilla Bb view they'll need to modify the `_removeElement` as directly after the plug `removeEl` the plain ole jquery call is going to happen still in the `view.remove`?